src/runner/JestRunner.ts

```diff
@@ -33,6 +33,7 @@ export class JestRunner extends TestRunner {
         testFiles.forEach((filename, index) => {
             this.reporter.onFileStart(filename);
             try {
+                resetTestData();
```

Comment:
Uh oh (unrelated to this PR).... I'm realizing when I migrated the test runner to Typescript instead of Brightscript, I removed the `_brs_.resetMocks()` call, and never added it back. Let me file an issue 😓

Reply:
Filed #89 for this!